### PR TITLE
Project enhancements

### PR DESCRIPTION
**Pull request by sgdc3**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2016 Gabriele C.
```
Comment (ljacqu) [inline LICENSE]: I see how it goes... ;)

**Comment by ljacqu** [comment]
Thank you!

File: LICENSE (hunk 1 above)
Comment (sgdc3) [inline LICENSE]: WTF DAMN YOU GITHUB

Comment (sgdc3) [inline LICENSE]: didn't see this sorry xD

**Pull request merged**